doc/api/n-api.md

```diff
@@ -5958,6 +5958,30 @@ idempotent.
 
 This API may only be called from the main thread.
 
+## Miscellaneous utilities
+
+## node_api_get_module_file_name
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+NAPI_EXTERN napi_status
+node_api_get_module_file_name(napi_env env, const char** result);
+
+```
+
+* `[in] env`: The environment that the API is invoked under.
+* `[out] result`: A null-terminated string containing the absolute path of the
```

Comment:
Is there any reason that it might be better  to make this a URL versus restricting to a path. For example are there any ES6 cases where a module might be loaded from a URL versus a file on disk?

Reply:
For the sake of future-proofing I will change it to "the absolute path of the location from which the add-on was loaded."

Reply:
Done.

Reply:
I guess that doesn't make it a full-fledged URL, complete with `file://`. Did you have *that* in mind?

Reply:
I was thinking something like `file://` to keep our options open. Not sure if there are any downsides?

Reply:
Aside from adding a little bit of parsing overhead, I can't think of any. I'll add that.

Reply:
Done.